stdlib/_typeshed/importlib.pyi

```diff
@@ -0,0 +1,18 @@
+# Implicit protocols used in importlib.
+# We intentionally omit deprecated and optional methods.
+
+from collections.abc import Sequence
+from importlib.machinery import ModuleSpec
+from types import ModuleType
+from typing import Protocol
+
+__all__ = ["LoaderProtocol", "MetaPathFinderProtocol", "PathEntryFinderProtocol"]
+
+class LoaderProtocol(Protocol):
+    def load_module(self, fullname: str, /) -> ModuleType: ...
+
+class MetaPathFinderProtocol(Protocol):
+    def find_spec(self, fullname: str, path: Sequence[str] | None, target: ModuleType | None = ..., /) -> ModuleSpec | None: ...
```

Comment:
I am not sure about making it positional only arguments... but I copied the definitions from `stdlib/sys/__init__.pyi`, `stdlib/importlib/abc.pyi`, `stdlib/types.pyi`.

Maybe it is better just to remove the `/`?

Reply:
Protocols should usually have positional-only arguments, because otherwise implementations need to use the exact same parameter names. Use named arguments only if consumers of the protocol pass keyword arguments.

Reply:
Oh, I see, that makes a lot of sense! Thank you very much for the clarification.

For consistency I also introduced https://github.com/python/typeshed/pull/11890/commits/6cddd52181c370e327d128f1e020e71fdaf1f892 to make the arguments in `PathEntryFinderProtocol` positional only.